experimental/packages/otlp-exporter-base/src/index.ts

```diff
@@ -13,7 +13,26 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-export * from './OTLPExporterBase';
-export * from './platform';
-export * from './types';
-export * from './util';
+export {
+  OTLPExporterNodeBase,
+  sendWithHttp,
+  createHttpAgent,
+  configureCompression,
+  OTLPExporterNodeConfigBase,
+  CompressionAlgorithm,
+  OTLPExporterBrowserBase,
+  sendWithXhr,
```

Comment:
Is it safe to re-export this with the `package.json` re-defining `./platform/index.js` as `./platform/browser/index.js`? All these exports won't be there then, right? :thinking: 

I tried packing and publishing these packages locally to a `verdaccio` registry that I use for testing, and I'm getting a lot of errors in the `opentelemetry-web` sample like:
```
Compiled with problems:

WARNING in ./node_modules/@opentelemetry/otlp-exporter-base/build/esm/index.js 16:0-164

export 'OTLPExporterNodeBase' (reexported as 'OTLPExporterNodeBase') was not found in './platform' (possible exports: OTLPExporterBrowserBase, sendWithXhr)


WARNING in ./node_modules/@opentelemetry/otlp-exporter-base/build/esm/index.js 16:0-164

export 'sendWithHttp' (reexported as 'sendWithHttp') was not found in './platform' (possible exports: OTLPExporterBrowserBase, sendWithXhr)


WARNING in ./node_modules/@opentelemetry/otlp-exporter-base/build/esm/index.js 16:0-164

export 'createHttpAgent' (reexported as 'createHttpAgent') was not found in './platform' (possible exports: OTLPExporterBrowserBase, sendWithXhr)


WARNING in ./node_modules/@opentelemetry/otlp-exporter-base/build/esm/index.js 16:0-164

export 'configureCompression' (reexported as 'configureCompression') was not found in './platform' (possible exports: OTLPExporterBrowserBase, sendWithXhr)


WARNING in ./node_modules/@opentelemetry/otlp-exporter-base/build/esm/index.js 16:0-164

export 'CompressionAlgorithm' (reexported as 'CompressionAlgorithm') was not found in './platform' (possible exports: OTLPExporterBrowserBase, sendWithXhr)
```

Reply:
Sorry, I had not tested this recently. I have fixed these errors now. What would be the way to catch these errors in examples during CI?

Reply:
No worries, thank you for following up on it :slightly_smiling_face: 

Unfortunately, there are no predefined ways to catch these errors in CI right now; I have wanted to work on this for a long time but have yet to get around to doing it. :/